voice/bxml/verbs/playAudio.md

```diff
@@ -39,7 +46,7 @@ None
 
 This shows how to use Bandwidth XML to play two audio clips into a phone call.
 
-⚠️ **ONLY** .wav files encoded as PCM or G711 and .mp3 files are supported.
+⚠️ **ONLY** `.wav` and `.mp3` files as described above are supported.
```

Comment:
The `s are not rendering well in the web page.

Reply:
Yeah it's gross, but I think I'd rather keep it and fix the formatting on the docsite by complaining to devx.